docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -98,10 +90,27 @@ Account contract implementation extending xref:ISRC6[`ISRC6`].
 
 * xref:#Account-\\__validate_declare__[`++__validate_declare__(self, class_hash)++`]
 
+.DeployableImpl
+
+* xref:#Account-\\__validate_deploy__[`++__validate_deploy__(self, hash, signature)++`]
+
 .PublicKeyImpl
 
-* xref:#Account-set_public_key[`++set_public_key(self, new_public_key)++`]
 * xref:#Account-get_public_key[`++get_public_key(self)++`]
+* xref:#Account-set_public_key[`++set_public_key(self, new_public_key)++`]
+--
+
+[.contract-index]
+.camelCase Support
+--
+.SRC6CamelOnlyImpl
+
```

Comment:
We are removing the support for SRC5Camel. Even if we haven't removed the methods, I don't think we should add them to docs. Anyway, in this case the component is not exposing even SRC5, instead of adding SRC5Camel, we need to remove the SRC5 apparition, and mention is a requirement (but not an embeddable implementation).